<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>430</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>186</red>
        <green>186</green>
        <blue>186</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>186</red>
        <green>186</green>
        <blue>186</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>186</red>
        <green>186</green>
        <blue>186</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>186</red>
        <green>186</green>
        <blue>186</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>AutoRotateScreen using APS v0.41</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/icon/Actions-transform-rotate.ico</normaloff>:/resource/icon/Actions-transform-rotate.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Orient0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string extracomment="Rotate Screen to Landscape mode"/>
    </property>
    <property name="text">
     <string>Orient 0 (0ยบ)</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScreenRotationDelay">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>226</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Screen Rotation Delay (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="SensorPollRate">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>277</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Sensor Poll Rate (ms):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Orient1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Orient 1 (90ยบ)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Orient3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>105</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Orient 3 (270ยบ)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ScreenRotationDelaySetting">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="minimum">
     <number>40</number>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Orient2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>205</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Orient 2 (180ยบ)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>About / Help</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="DataTable">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>2</y>
      <width>376</width>
      <height>157</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>46</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>46</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>26</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>26</number>
    </attribute>
    <row>
     <property name="text">
      <string>0 (Landscape)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1 (Portrait)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2 (Landscape Flipped)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3 (Portrait Flipped)</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Calibrating Orient:</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Enable</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MinX</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MaxX</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MinY</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MaxY</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="AutoRotationButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>238</red>
          <green>238</green>
          <blue>238</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>238</red>
          <green>238</green>
          <blue>238</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>AutoRotation ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveSettingsButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Save Current Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataSaveStatus">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Unsaved Data : Nill</string>
    </property>
   </widget>
   <widget class="QLabel" name="AutoRotationStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Auto Rotation Status: Nill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalibrationButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>238</red>
          <green>238</green>
          <blue>238</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>238</red>
          <green>238</green>
          <blue>238</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Calibrate Current Orientation</string>
    </property>
   </widget>
   <widget class="QLabel" name="CalibrationStatus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Calibration-in-progress: Nill</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SensorPollRateSetting">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>90000000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="RotationMethodOSRadio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Native OS Rotation API (default)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RotationMethodKeyBShortRadio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Keyboard Shortcut</string>
    </property>
   </widget>
   <widget class="QLabel" name="RotationMethodLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation Method:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="StatusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>showHelp()</slot>
 </slots>
</ui>
